.env.template

```diff
@@ -61,6 +61,15 @@ GOOGLE_SECRET=
 GITHUB_KEY=
 GITHUB_SECRET=
 
+ALLOWED_SERVICES:
```

Comment:
This should not be part of the env I think, but rather be set explicitly in the Dockerfile.

Currently it's here:
https://github.com/datahq/plans/blob/master/auth-docker/Dockerfile

But we should probably move this Dockerfile to its own repository.